File: app/src/main/java/com/example/xc_m6_java/activity/MainActivity.java
package com.example.xc_m6_java.activity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.example.xc_m6_java.R;
import com.example.xc_m6_java.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        initViews();
    }

    private void initViews() {
        binding.btnLanguageActivity.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                callLanguageActivity();
            }
        });

        binding.btnSharedPreferenceActivity.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                callSharedPreferenceActivity();
            }
        });

        binding.bDatabaseActivity.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                callDatabaseActivity();
            }
        });
    }

    private void callDatabaseActivity() {
        Intent intent = new Intent(this,DatabaseActivity.class);
        startActivity(intent);
    }

    private void callSharedPreferenceActivity() {
        Intent intent = new Intent(this,PrefsActivity.class);
        startActivity(intent);
    }

    private void callLanguageActivity() {
        Intent intent = new Intent(this,LanguageActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/example/xc_m6/activity/MainActivity.kt
package com.example.xc_m6.activity

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.xc_m6.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initViews()
    }

    private fun initViews() {
        btn_languageActivity.setOnClickListener { callLanguageActivity() }

        b_PreferenceActivity.setOnClickListener { callSharedPreferenceActivity() }

        b_databaseActivity.setOnClickListener { callDatabaseActivity() }
    }

    private fun callDatabaseActivity() {
        val intent = Intent(this,DatabaseActivity::class.java)
        startActivity(intent)
    }

    private fun callSharedPreferenceActivity() {
        val intent = Intent(this,PreferenceActivity::class.java)
        startActivity(intent)
    }

    private fun callLanguageActivity() {
        var intent = Intent(this,LanguageActivity::class.java)
        startActivity(intent)
    }
}